docker-compose-dev.yml

```diff
@@ -28,7 +28,6 @@ services:
       - DISABLE_UPNP=
     networks:
```

Comment:
Should we remove whole network section? In case stuff breaks at the point dncore_network doesn't exist this will break.

Reply:
yes! we must